usaspending_api/awards/migrations/0051_auto_20190425_1643.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-04-25 16:43
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('awards', '0050_remove_financialaccountsbyawards_award_type'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='transactionfabs',
+            name='unique_award_key',
+            field=models.TextField(db_index=True, null=True),
```

Comment:
Is there a way to have this enforce uniqueness?

Reply:
These are unique at the award level, not the transaction level.